pkg/dashboards/istio/istio_service.go

```diff
@@ -0,0 +1,238 @@
+				markdownPanel.Text("<div class=\"dashboard-header text-center\">\n<span>CLIENT WORKLOADS</span>\n</div>"),
+			),
+		),
+		panels.IncomingRequestsByClient(datasource, labelMatcher),
+		panels.IncomingSuccessRateByClient(datasource, labelMatcher),
+	)
+}
+
+// Client Workloads (II) section
+func withClientWorkloadsIISection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Client Workloads (II)",
+		panelgroup.PanelsPerLine(3),
+		panelgroup.PanelHeight(6),
+		panels.IncomingRequestDurationByClient(datasource, labelMatcher),
+		panels.IncomingRequestSizeByClient(datasource, labelMatcher),
+		panels.ResponseSizeByClient(datasource, labelMatcher),
+	)
+}
+
+// Client Workloads (III) section
+func withClientWorkloadsIIISection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Client Workloads (III)",
+		panelgroup.PanelsPerLine(2),
+		panelgroup.PanelHeight(6),
+		panels.BytesReceivedFromTCPClient(datasource, labelMatcher),
+		panels.BytesSentToTCPClient(datasource, labelMatcher),
+	)
+}
+
+// Service Workloads section
+func withServiceWorkloadsSection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Service Workloads",
+		panelgroup.PanelsPerLine(2),
+		panelgroup.PanelHeight(6),
+		// Header panel
+		panelgroup.AddPanel("SERVICE WORKLOADS",
+			markdownPanel.Markdown("Service Workloads Header",
+				markdownPanel.Text("<div class=\"dashboard-header text-center\">\n<span>SERVICE WORKLOADS</span>\n</div>"),
+			),
+		),
+		panels.IncomingRequestsByService(datasource, labelMatcher),
+		panels.IncomingSuccessRateByService(datasource, labelMatcher),
+	)
+}
+
+// Service Workloads (II) section
+func withServiceWorkloadsIISection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Service Workloads (II)",
+		panelgroup.PanelsPerLine(3),
+		panelgroup.PanelHeight(6),
+		panels.IncomingRequestDurationByService(datasource, labelMatcher),
+		panels.IncomingRequestSizeByService(datasource, labelMatcher),
+		panels.ResponseSizeByService(datasource, labelMatcher),
+	)
+}
+
+// Service Workloads (III) section
+func withServiceWorkloadsIIISection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Service Workloads (III)",
+		panelgroup.PanelsPerLine(2),
+		panelgroup.PanelHeight(6),
+		panels.BytesReceivedFromTCPService(datasource, labelMatcher),
+		panels.BytesSentToTCPService(datasource, labelMatcher),
+	)
+}
+
+func BuildIstioService(project string, datasource string, clusterLabelName string) dashboards.DashboardResult {
+	clusterLabelMatcher := dashboards.GetClusterLabelMatcher(clusterLabelName)
+	return dashboards.NewDashboardResult(
+		dashboard.New("istio-service-dashboard",
+			dashboard.ProjectName(project),
+			dashboard.Name("Istio Service Dashboard"),
```

Comment:
Also maybe we can name the dashboards in a similar way as the kuberntes ones, for example https://github.com/perses/community-dashboards/blob/main/pkg/dashboards/kubernetes/kubelet/kubelet.go#L162 as `Istio / Service Dashboard`,` Istio /Workload Dashboard`, to keep consistent with the existing ones. WDYT?

<img width="478" height="361" alt="Screenshot 2025-08-08 at 15 29 17" src="https://github.com/user-attachments/assets/4a5571ab-e7a5-4291-9155-9029bd011fd5" />